bin/templates/templates.xml

```diff
@@ -151,7 +150,23 @@
         <h2>Useful link</h2>
             <ul>
                 <li><a href="http://jmeter.apache.org/usermanual/build-web-test-plan.html" >http://jmeter.apache.org/usermanual/build-web-test-plan.html</a></li>
-            </ul>
+            </ul>       
+        ]]></description>
+    </template>
+	<template isTestPlan="true">
+        <name>Functional Testing Test Plan [01]</name>
+        <fileName>/bin/templates/functional-testing-01-test-plan.jmx</fileName>
+        <description><![CDATA[
+        <h1>Building a Functional Testing Test Plan [01]</h1>
+		<p>You can use JMeter to automate API Testing</p>
+		<p>This template of Test Plan contains first step when starting creating JMeter script</p>
+        <h2>Useful links</h2>
+            <ul>
+                <li><a href="http://jmeter.apache.org/usermanual/test_plan.html" >http://jmeter.apache.org/usermanual/test_plan.html</a></li>
```

Comment:
Even if this is probably a copy and paste thing, try to use `https://...` in URLs, when possible.